src/save_index.hpp

```diff
@@ -124,6 +124,8 @@ class save_index_class : public std::enable_shared_from_this<save_index_class>
 	config& data();
 
 	static void fix_leader_image_path(config& data);
+	/** Deletes non-existent save files from the index **/
+	void clean_up_index(void);
```

Comment:
What's the difference between `void clean_up_index(void);` and `void clean_up_index();` given this is C++ rather than C?